pom.xml

```diff
@@ -199,7 +199,7 @@
                                     <!-- TODO: uncomment for checking dependency conflicts -->
                                     <!-- <DependencyConvergence/> -->
                                     <requireJavaVersion>
-                                        <version>[1.8,12)</version>
+                                        <version>[11,)</version>
```

Comment:
@imbajin Do we enforce a specific Java version here, abandoning Java 8?

Reply:
> @imbajin Do we enforce a specific Java version here, abandoning Java 8?

In my view, Yes (we could just define it in the root pom?)